packages/core/src/shared/utilities/messages.ts

```diff
@@ -139,6 +140,43 @@ export async function showConfirmationMessage({
     }
 }
 
+/**
+ * Shows a prompt for the user to reauthenticate
+ *
+ * @param message the line informing the user that they need to reauthenticate
+ * @param connect the text to display on the "connect" button
+ * @param doNotShow the text to display on the "do not show again" button
+ * @param suppressId the ID of the prompt in
+ * @param reauthFunc the function called if the "connect" button is clicked
+ */
+export async function showReauthenticateMessage({
+    message,
+    connect,
+    doNotShow,
+    suppressId,
+    reauthFunc,
+}: {
+    message: string
+    connect: string
+    doNotShow: string
```

Comment:
This should be the same throughout the extension. CW doesn't need its own "do not show" text.

I'll do this and possibly some other things in a followup PR.

Reply:
Followup: https://github.com/aws/aws-toolkit-vscode/pull/4679